3]. Competitive Programming/HackerRank/C++/Arrays Introduction.cpp

```diff
@@ -0,0 +1,20 @@
+#include <cmath>
```

Comment:
Can You add the HackerRank C++ problem in folder
![image](https://user-images.githubusercontent.com/60146338/122511029-4205ab80-d024-11eb-98da-e89c8059de3f.png)

and put the number in front of the question like  `_01)_Hello_World.cpp` in

Reply:
Yeah Sure, no problem. I haven't completed all the problems but I will do it ASAP.